### DISCUSSION TITLE
Many-Many relation in Lucid ORM

### DISCUSSION THREAD

**Discussion by naveenbasyal** [community/community#136442] [Programming Help] [answered]
i was getting stuck in implementing many to many relationship in lucid model in adonijs v6, any help would be appreciated !!

**Comment by PuneetSharma52** [comment, marked as answer]
yes you can use the code snippet written below: 
`@manyToMany(() => Skill, {
  localKey: 'id',
  pivotForeignKey: 'user_id',
  relatedKey: 'id',
  pivotRelatedForeignKey: 'skill_id',
})
declare skills: ManyToMany<typeof Skill>`
This code you have to implement it in both your models and no need to create model for your pivot table, I hope this will be helpful✌🏻

**Reply by naveenbasyal** [reply]
Thank you , really saved much time !!
